scania-digital-design-system/sdds
# fix(card): improvement to the card components storybook

**Describe pull-request**  
Improving the story of the card component and rebase agains main. 

**Solving issue**  
Fixes: -

**How to test**  
1. Go to storybook link below
2. Check in components -> card
3. Test out the controls and see that the design is aligned with the one in Figma. 

**Screenshots**  
-

**Additional context**  
-
